README.md

```diff
@@ -4,9 +4,8 @@ This repository contains scripts for packaging `jenkins.war` into various platfo
 The following platforms are currently supported:
 
   * Windows MSI: `msi/`
-  * RedHat/CentOS RPM: `rpm/`
+  * RedHat/CentOS/OpenSuSE RPM: `rpm/`
```

Comment:
According to https://www.opensuse.org/ this is spelled "openSUSE":

```suggestion
  * RedHat/CentOS/openSUSE RPM: `rpm/`
```

Reply:
Last time I used this distribution it was still called "S.u.S.E." ("Software und SystemEntwicklung") back in '97 :D